test/test_normalize.js

```diff
@@ -0,0 +1,225 @@
+const test = require('ava')
+const normalizeSchema = require('../lib/normalize.js').normalizeSchema
+const normalizeType = require('../lib/normalize.js').normalizeType
+const nomralizeDateFormat = require('../lib/normalize.js').nomralizeDateFormat
+const normalizeResourceName = require('../lib/normalize.js').normalizeResourceName
+const normalizeAll = require('../lib/normalize.js').normalizeAll
+
+let dp = {
+  "name": "example",
+  "licenses": {
+    "name": "example license",
+    "url": "https://example/license.com"
+  },
+  "resources": [
+  {
+    "name": "PPP-Gdp",
+    "path": "dp/ppp-gdp.csv",
+    "format": "csv",
+    "mediatype": "text/csv",
+    "schema": {
+    "fields": [
+      {
+        "name": "Country",
+        "type": "string"
+      },
+      {
+        "name": "Country ID",
+        "type": "decimal",
+        "description": "ISO 3166-1 alpha-2 code"
+      },
+      {
+        "name": "Year",
+        "type": "date",
+        "format": "YYYY",
+        "description": "Relevant year"
+      }
+    ]}
+  }]
+}
+
+test('checks normalized schema', t => {
+  
+  let res = normalizeSchema(dp)
+  let exp = {
```

Comment:
this is minor but let's keep to convention - let is for variables that you are going to modify, const for variables that are constant (assigned once and never touched - like this one)
this one is `const`